.github/workflows/cdk-nag-workflow.yml

```diff
@@ -0,0 +1,20 @@
+# Workflow that runs unit test
+name: CDK Nag Test
+
+on: [push, pull_request]
+
+jobs:
+  cdk-nag:
+    name: CDK Nag Check
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [16.x]
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node-version }}
+      - run: |
+          cd source/infrastructure && npm i
```

Comment:
To avoid having to `cd`, you can specify:

```
defaults:
      run:
        working-directory: source
```

Also, use [npm ci](https://docs.npmjs.com/cli/v9/commands/npm-ci) for the following reasons:
- it will fail if package-lock is missing
- it will fail if package-lock and package.json are not reconcile-able - `npm i` may resolve differences by modifying package or package-lock, so package-lock does not reflect what was actually used in the workflow

Reply:
good catch. Will have to update the release guidance doc with this as well as I pretty much copied-pasted exactly what was in them for this